coreservices/partsrelationshipservice/dev/connector-integration-test.sh

```diff
@@ -9,10 +9,11 @@ curl -O https://raw.githubusercontent.com/kadwanev/retry/master/retry
 chmod +x retry
 ./wait-for-it.sh -t 60 provider:8181
 ./wait-for-it.sh -t 60 consumer:9191
+./wait-for-it.sh -t 60 prs:8080
 mkdir -p /tmp/copy/source /tmp/copy/dest
-echo value-in-test-document > /tmp/copy/source/test-document.txt
-requestId=$(curl -f -X POST http://consumer:9191/api/file -H "Content-type:application/json" -d '{"filename":"test-document", "connectorAddress": "http://provider:8181", "destinationPath":"/tmp/copy/dest/new-document.txt"}')
+requestId=$(curl -f -X POST http://consumer:9191/api/file -H "Content-type:application/json" -d '{"connectorAddress": "http://provider:8181", "destinationPath":"/tmp/copy/dest/new-document.txt", "partsTreeRequest": {
+                "oneIDManufacturer": "CAXSWPFTJQEVZNZZ", "objectIDManufacturer": "UVVZI9PKX5D37RFUB", "view": "AS_BUILT", "aspect": "MATERIAL", "depth": 2}}')
 ./retry -s 1 -t 120 "test \$(curl -f http://consumer:9191/api/datarequest/$requestId/state) == COMPLETED"
 curl -f http://consumer:9191/api/datarequest/$requestId/state
 echo
-cat /tmp/copy/dest/new-document.txt
+cat /tmp/copy/dest/new-document.txt | grep "relationships\":\[\]"
```

Comment:
```suggestion
cat /tmp/copy/dest/new-document.txt
grep "relationships\":\[\]" /tmp/copy/dest/new-document.txt
```

leaving the cat does not hurt, for the user to see